server/services/zigbee2mqtt/lib/getPermitJoin.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * @description Prepares service and starts connection with broker.
+ * @returns {boolean} Status of Zigbee2mqtt Permit join.
+ * @example
+ * init();
+ */
+function getPermitJoin() {
+  return this.z2mPermitJoin;
```

Comment:
Here you don't `Prepares service and starts connection with broker`
It's only a getter.

Reply:
Documentation updated